.gitignore

```diff
@@ -401,4 +401,7 @@ $RECYCLE.BIN/
 # Jest Testing Snapshots
 browser/src/__snapshots__
 
+# Images folder
+api/images
```

Comment:
Much better - you may have to delete the folder, since it's still there with all the images uploaded to it